packages/stage-ui/src/components/Scenes/VRM/Model.vue

```diff
@@ -1,8 +1,11 @@
 <script setup lang="ts">
 import type { VRMCore } from '@pixiv/three-vrm-core'
+import type { AnimationClip } from 'three'
 
 import { VRMUtils } from '@pixiv/three-vrm'
+import { useVRM } from '@proj-airi/stage-ui/stores'
```

Comment:
```suggestion
import { useVRM } from '../../../stores'
```

Importing `stage-ui` itself inside `stage-ui` will have unexpected consequences. Let's use `../../../stores`